app/src/screens/settings/ManageDocumentsScreen.tsx

```diff
@@ -314,11 +314,9 @@ const ManageDocumentsScreen: React.FC = () => {
             <PrimaryButton onPress={handleScanDocument}>
               Scan New ID Document
             </PrimaryButton>
-            {__DEV__ && (
-              <SecondaryButton onPress={handleGenerateMock}>
-                Generate Mock Document
-              </SecondaryButton>
-            )}
+            <SecondaryButton onPress={handleGenerateMock}>
```

Comment:
fyi @remicolin @aaronmgdr this fix only made it into the android version of `2.6.4`. iOS was already submitted